hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,22 @@
+RELEASE_TYPE: patch
```

Comment:
arguably should be `minor` instead?

Reply:
Either is fine, but with no interface change I'm inclined to go with `patch`.